components/log-viewer-webui/client/src/components/QueryBox/index.tsx

```diff
@@ -1,14 +1,54 @@
 import {
+    Button,
     Input,
     InputProps,
     Progress,
     theme,
+    Tooltip,
+    Typography,
 } from "antd";
 import {Nullable} from "src/typings/common";
 
 import styles from "./index.module.css";
 
-export interface QueryBoxProps extends InputProps {
+
+export interface CaseSensitiveToggleProps {
```

Comment:
not sure why the liner doesn't catch this -

we tend to put all exports towards the end of sources. e.g.,
```
interface CaseSensitiveToggleProps {...}

...


export {CaseSensitiveToggleProps};
```

@coderabbitai create an issue to track why linter didn't catch this.